### ISSUE TITLE
Need to update MPCs for VS2019

### ISSUE THREAD

**Issue by grossag** [MicrosoftDocs/visualstudio-docs#3035]
The MPCs listed in this page (e.g. 08860, 08862, and 08866) are for VS2017; VS2019 uses different MPCs:

1. Enterprise may be 09260 (I haven't confirmed it).
2. Professional is 09262 (I have confirmed it).
3. Test Professional could be 09266, but DDConfigCA.exe doesn't actually create that registry value so I am doubting this and haven't confirmed it.

In #3, what I mean is that when run elevated with no command-line params, DDConfigCA.exe from VS2019 GA creates the following registry keys under HKCR\Classes\Licenses\41717607-F34E-432C-A138-A3CFD7E25CDA: 09260, 09262, 09263, 09264, 09265, 09271, 09275, 09276, 09277, 09278, 09279, 09280, and 0bcad. I would expect DDConfigCA.exe to create registry keys related to all supported MPCs, but I don't see 09266, so I don't know what the Test Professional MPC is.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 11e2b93a-e89d-42ce-23d7-4cf85f47ba25
* Version Independent ID: 4083c0d5-c871-ef24-d11c-b6ec1adbb571
* Content: [Automatically apply product keys - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/install/automatically-apply-product-keys-when-deploying-visual-studio?view=vs-2019#feedback)
* Content Source: [docs/install/automatically-apply-product-keys-when-deploying-visual-studio.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/install/automatically-apply-product-keys-when-deploying-visual-studio.md)
* Product: **visual-studio-windows**
* Technology: **vs-installation**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by TerryGLee** [comment]
@grossag Thanks for letting us know; we have a PR in place to change the MPC values on the page. Meanwhile, you are correct in stating the following:

Visual Studio Enterprise | 09260
Visual Studio Professional | 09262

Thanks again for your feedback! We appreciate it.

**Comment by grossag** [comment]
Thanks @TerryGLee . Is there a MPC for Test Professional? Or is that going away?

**Comment by TerryGLee** [comment]
@grossag Thanks for your feedback. We've updated the documentation with the correct codes. As to your question about Test Professional -- there isn't an MPC for that now, but here's some further information about our testing services:

Azure DevOps Services and Azure DevOps Server offer a web-based solution for manual testing that is included in the Visual Studio Test Professional subscription, known as Azure Test Plans. Azure Test Plans is a web-based manual testing solution, which works across all platforms and with all browsers. It provides experiences across Plan, Author, Execute and Track stages of Manual testing, for web-based and desktop apps.
 
In the past, we have also shipped a client based solution as Microsoft Test Manager (MTM). Microsoft Test Manager client is a desktop-based manual testing solution, which has traditionally been used by testers for their manual testing needs.  For more information on the support lifecycle of Microsoft Test Manager client and the benefits Azure Test Plans provides over MTM, read this topic: [Guidance on Microsoft Test Manager usage](https://docs.microsoft.com/en-us/azure/devops/test/mtm/guidance-mtm-usage?view=azure-devops).  
 
Moving forward, the Microsoft Test Manager client will be deprecated and Visual Studio Test Professional subscription will continue to grant access to the web-based solution for test management that is known as Azure Test Plans.
